hw06-solid/src/main/java/ATM/CashRegister.java

```diff
@@ -0,0 +1,77 @@
+package ATM;
+
+import java.util.*;
+
+public class CashRegister {
+    private final ArrayList<Cell> cellBlock = new ArrayList<>();
```

Comment:
Тип лучше объявлять интерфейсом
```suggestion
    private final List<Cell> cellBlock = new ArrayList<>();
```